CONTRIBUTING.md

```diff
@@ -0,0 +1,44 @@
+# Contributing to the Optimizely Starter Kit
+
+We welcome contributions and feedback! All contributors must sign our [Contributor License Agreement (CLA)](https://docs.google.com/a/optimizely.com/forms/d/e/1FAIpQLSf9cbouWptIpMgukAKZZOIAhafvjFCV8hS00XJLWQnWDFtwtA/viewform) to be eligible to contribute. Please read the [README](README.md) to set up your development environment, then read the guidelines below for information on submitting your code.
+
+## Development process
+
+1. Fork the repository and create your branch from master.
+2. Please follow the [commit message guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines) for each commit message.
+3. Update relevant `CHANGELOG.md` if users should know about this change.
+4. `git push` your changes to GitHub.
+5. Open a PR from your fork into the master branch of the original repo
+6. Make sure that there are no merge conflicts between your branch and `master`.
+7. Open a pull request from `YOUR_NAME/branch_name` to `master`.
+8. A repository maintainer will review your pull request and, if all goes well, squash and merge it!
+
+Note: Please don't change the `package.json` or `VERSION`. We'll take care of bumping the version when we next release.
+
+## License
+
+All contributions are under the CLA mentioned above. For this project, Optimizely uses the Apache 2.0 license, and so asks that by contributing your code, you agree to license your contribution under the terms of the [Apache License v2.0](http://www.apache.org/licenses/LICENSE-2.0). Your contributions should also include the following header:
+
+```
+/****************************************************************************
+ * Copyright YEAR, Optimizely, Inc. and contributors                        *
+ *                                                                          *
+ * Licensed under the Apache License, Version 2.0 (the "License");          *
+ * you may not use this file except in compliance with the License.         *
+ * You may obtain a copy of the License at                                  *
+ *                                                                          *
+ *    http://www.apache.org/licenses/LICENSE-2.0                            *
```

Comment:
I think `https://` in this message might be better.

Reply:
Updated!